boards/DCM/Src/App/states/App_AllStates.c

```diff
@@ -14,21 +14,14 @@ static void App_SendAndReceiveHeartbeat(struct HeartbeatMonitor *hb_monitor)
     // JSONCAN -> App_CanTx_SetPeriodicSignal_MISSING_HEARTBEAT(can_tx, is_missing_hb);
 }
 
-void App_AllStatesRunOnTick1Hz(struct StateMachine *const state_machine)
-{
-    struct DcmWorld *      world            = App_SharedStateMachine_GetWorld(state_machine);
-    struct RgbLedSequence *rgb_led_sequence = App_DcmWorld_GetRgbLedSequence(world);
-
-    App_SharedRgbLedSequence_Tick(rgb_led_sequence);
-}
+void App_AllStatesRunOnTick1Hz(struct StateMachine *const state_machine) {}
```

Comment:
Could make it 'UNUSED(state_machine);' or do we need this function?

Reply:
I'll make it UNUSED, I feel like we might an all states 1Hz function later